src/hooks/useTimeout.js

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+
+const useTimeout = ( callback, delay ) => {
+  const [ timeoutId, setTimeoutId ] = useState( -1 );
```

Comment:
nie wystarczy tu 0?

Reply:
Dla bezpieczeństwa wolę dać -1 bo to w sumie id jest a indeks może być równy 0